File: examples/language-test-app/Python/Utilities.py
from zsharp import export


@export
def greet(name: str) -> str:
    return f"Hello from Python, {name or 'Z# developer'}!"


@export
def add(left: int, right: int) -> int:
    return left + right


@export
def is_ready() -> bool:
    return True

File: examples/language-test-app/Lua/Utilities.lua
local utilities = {}

function utilities.greet(name)
  if name == "" then
    name = "Z# developer"
  end
  return "Hello from Lua, " .. name .. "!"
end

function utilities.add(left, right)
  return left + right
end

function utilities.is_ready()
  return true
end

return utilities
